zerver/tornado/descriptors.py

```diff
@@ -1,7 +1,7 @@
 from typing import Dict, Optional
 
 if False:
-    from zerver.tornado.event_queue import ClientDescriptor
+    from zerver.tornado.event_queue import ClientDescriptor  # type: ignore # TODO: Perhaps explain this.
```

Comment:
Does this also have to do with cyclic dependencies?

Reply:
This should also use [`if TYPE_CHECKING`](https://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles).